game/src/sandbox/misc_tools.rs

```diff
@@ -30,24 +30,28 @@ impl RoutePreview {
             .and_then(|id| id.agent_id())
         {
             let now = app.primary.sim.time();
+            let zoomed = ctx.canvas.cam_zoom >= app.opts.min_zoom_for_detail;
             if self
                 .preview
                 .as_ref()
-                .map(|(a, t, _)| agent != *a || now != *t)
+                .map(|(a, t, z, _)| agent != *a || now != *t || zoomed != *z)
                 .unwrap_or(true)
             {
-                if let Some(trace) = app.primary.sim.trace_route(agent, &app.primary.map, None) {
-                    let mut batch = GeomBatch::new();
-                    batch.extend(
-                        app.cs.route,
-                        trace.dashed_lines(
-                            Distance::meters(0.75),
-                            Distance::meters(1.0),
-                            Distance::meters(0.4),
-                        ),
-                    );
-                    self.preview = Some((agent, now, batch.upload(ctx)));
+                let mut batch = GeomBatch::new();
+                if zoomed {
```

Comment:
So currently we draw the map whenever an agent is clicked, but now that you can click while zoomed out we don't want to draw the route for the unzoomed agent. Am I interpreting this code correctly? 

Why wouldn't we want to draw the route for unzoomed agents?

Reply:
You're correct. I think it looks a bit too busy. While unzoomed, you can click on one agent, then mouseover another. Drawing routes for both looks disorienting. (You can also do this while zoomed in, but it seems less overwhelming there)

Reply:
Ah, I forgot that routes showed while hovered when zoomed in. It makes sense to not show both when zoomed out.